src/Packetizer/PassthroughPacketizer.h

```diff
@@ -0,0 +1,96 @@
+/**@file
+ * This file is part of the TASTE Runtime Common.
+ *
+ * @copyright 2022 N7 Space Sp. z o.o.
+ *
+ * TASTE Runtime Common was developed under a programme of,
+ * and funded by, the European Space Agency (the "ESA").
+ *
+ * Licensed under the ESA Public License (ESA-PL) Permissive,
+ * Version 2.3 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://essr.esa.int/license/list
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef PASSTHROUGH_PACKETIZER_H
+#define PASSTHROUGH_PACKETIZER_H
+
+/**
+ * @file    PassthroughPacketizer.h
+ * @brief   Main header file for Space Packet passthrough packetizer.
+ */
+
+#include "Packetizer.h"
+
+/**
+ * @brief   Initialize Packetizer struct
+ *
+ * @param[in]   self         Pointer to a structure representing Packetizer
+ * @param[in]   busId        Bus ID of the device
+ * @param[out]  headerSize   Primary header size of the Packetizer
+ */
+void PassthroughPacketizer_init(Packetizer* const self, const enum SystemBus busId, size_t* const headerSize);
+
+/**
+ * @brief   Packetize given data with Space Packet header and CRC.
```

Comment:
CRC? Space Packet? Mention something that this is a dummy function which in practice does nothing.

Reply:
changed